news/2021-06-20-mappers-report-may.md

```diff
@@ -0,0 +1,160 @@
+
+### osu!flushed Taiko Mapping Contest
+
+The [osu!flushed Taiko Mapping Contest](https://osu.ppy.sh/community/forums/topics/1342681) is a brand new contest where the responsibility of choosing the song falls onto your hands. That's right, you will have an *almost* complete freedom in song choice, where the only restriction is that it needs to follow a certain theme.
+
+As for the prizes, the winner gets 6 months of osu!supporter, while the runners-ups of 2nd and 3rd place receive 4 months of osu!supporter.
+
+Don't wait for the drums to roll - go ahead and sign yourself up via the thread above to show who places the blues and reds the best!
+
+### osu!catch 2021 Summer Featured Artist Cup
+
+As the summer cracks the door open, the fruits start ripening. To fill your cup with the cream of the crop, be sure to join the [osu!catch Featured Artist Cup](https://osu.ppy.sh/community/forums/topics/1327798) right away!
+
+Featuring songs from our [Featured Artists](https://osu.ppy.sh/beatmaps/artists) who joined the lineup between March 20th - June 20th, this contest will test your mapping capabilities in two difficulties: one being Platter or lower, and the other being Rain or higher.
+
+But that's not all - this one has some hefty prizes too! 
+
+The winner will be rewarded with 100 USD in cash, plus having their set modded by the host and the judges! Should you not make it to the top, the runners-up will still get 20 USD in cash in addition to 4 months of osu!supporter for 2nd place, and the 3rd place will enjoy 2 months of osu!supporter.
+
+Have we got you interested in getting your hands dirty with the harvest? Then be quick - the sign-ups are closing soon on **June 20th**!
+
+### osu!catch Mapping World Cup 2021
+
+Brought to you by the osu!catch mapping grand-master [Ascendance](https://osu.ppy.sh/users/2931883) for the third time, the [osu!catch Mapping World Cup 2021](https://osu.ppy.sh/community/forums/topics/1350409) will have the best mappers from all over the globe gather around and have an intense competition that will determine the next osu!catch Mapping Champions!
+
+Needless to say, such a contest is not going to be without prizes but all the details on that matter are currently work-in-progress. To keep track of the updates and learn more about the contest, we highly suggest you to check the [forum thread](https://osu.ppy.sh/community/forums/topics/1350409), and dive right into the [o!CMWC Discord server](https://discord.gg/eTUFAjf "Discord").
+
+Alright, osu!catch folks! Hopefully, we've made your day with these two majestic contests, but make sure to compete responsibly, you wouldn't want to... *overdose* on fruit juice after all, would you?
```

Comment:
```suggestion
Alright, osu!catch folks! Hopefully we've made your day with these two majestic contests. However, make sure to compete responsibly, since you wouldn't want to... *overdose* on fruit juice after all, would you?
```

Reply:
Pretty sure that comma is necessary.

There is no need to separate the sentence and start with "however," given that the tone should be informal. No need to change anything here.

Reply:
I suggested to split the sentence because there were many commas, but alright. The first comma isn't necessary but can be there.

The "since" prevents comma splice, though.

Reply:
Added that "since" but otherwise it reads completely fine to me.